src/content/docs/security/security-privacy/compliance/fedramp-compliant-endpoints.mdx

```diff
@@ -426,12 +487,26 @@ To ensure FedRAMP compliance, all traffic reporting to `insights-collector.newre
   </tbody>
 </table>
 
-## Browser monitoring [#browser-endpoints]
+### Log API (#log-api)
 
-Our browser monitoring agent transmits data to New Relic's [collectors](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/glossary#collector) by using the domain `gov-bam.nr-data.net` for both `beacon` and `error_beacon`.
+To ensure FedRAMP compliance for data sent via the [Log API](/docs/logs/log-management/log-api/introduction-log-api/), the solution for almost all our logging tools is to replace the `https://log-api.newrelic.com/log/v1` endpoint with `https://gov-log-api.newrelic.com/log/v1`. Here are details for various tools:
 
-Ensure the script element tag that contains configuration and timing data is updated to the following:
+#### Log forwarders
 
-```
-window.NREUM||(NREUM={});NREUM.info={"beacon":"gov-bam.nr-data.net","errorBeacon":"gov-bam.nr-data.net"...
-```
+Here are details on changing the endpoint for our [log forwarders](/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic): 
+
+* AWS Firelens: Add the `endpoint` property to the `options` field of the `logConfiguration`, similar to to the EU account endpoint change shown in [these Firelens endpoint configuration instructions](/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/aws-firelens-plugin-log-forwarding#find-data).
+* Fluentbit: Use [our Fluentbit endpoint configuration](https://github.com/newrelic/newrelic-fluent-bit-output#configuration-parameters).
+* Fluentd: Use [our Fluentd endpoint instructions](/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/fluentd-plugin-log-forwarding#instrument-plugin).
+* Infrastructure agent: See [FedRAMP for infrastructure](/#infrastructure-endpoints).
+* Kubernetes: Our Kubernetes integration logs are based on fluentbit’s output plugin. Use [these endpoint instructions](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging#installation).
+* Logstash: Use [our Logstash endpoint configuration](/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/logstash-plugin-log-forwarding#instrument-plugin).
+* Syslog: For configuring syslog clients, see [TCP endpoint configuration](/docs/logs/log-management/log-api/use-tcp-endpoint-forward-logs-new-relic/).
+* S3: Not yet available.
+* Vector: Not yet available.
+
+To use the Log API directly, you'd edit the [the Log API endpoint configuration](/docs/logs/log-management/log-api/introduction-log-api#endpoint).
```

Comment:
either remove the 'the' after you'd edit or delete it from the link.  so you don't get "the the"